validation/td-validation.ttl

```diff
@@ -170,26 +170,26 @@
     ] ;
     sh:property [
         sh:path td:schemaDefinitions ;
-        skos:definition """Set of named data schemas.
-                To be used in a <code>schema</code>
-                name-value pair inside an 
-                <code>AdditionalExpectedResponse</code> object."""^^rdf:HTML ;
+        skos:definition """Set of global-based named data schemas.
+                To be used in a local <code>schema</code>
+                name-value pair such as inside an 
+                <code>ExpectedResponse</code> and <code>AdditionalExpectedResponse</code> object."""^^rdf:HTML ;
         sh:node :DataSchemaShape ;
         sh:order 18 ;
     ] ;
     skos:scopeNote """<p><span class="rfc2119-assertion" id=
           "td-context-ns-thing-mandatory">The <code>@context</code>
           name-value pair <em class="rfc2119" title=
           "MUST">MUST</em> contain the anyURI
-          <code>https://www.w3.org/2019/wot/td/v1</code> either
+          <code>https://www.w3.org/2022/wot/td/v1.1</code> either
```

Comment:
That seems to be an unrelated change, which should be in a separate PR.

Reply:
strange. It seems a race condition with other PRs. The namespace https://www.w3.org/2022/wot/td/v1.1 is correct